andela/wg-wits
# #159236193 Rework landing page

#### What does this PR do?
* Redesigns the Landing page to give it more value.

#### Description of Task to be completed?
* Make a rework on the landing page so that a user understands the features provided by wger before scrolling.
* Make the landing page appealing to both males and females

#### How should this be manually tested?
* Clone the project, ```https://github.com/andela/wg-wits.git```.
* In your terminal, ```cd wg-wits```
* Checkout to the ft-landing-page-rework-159236193 branch.
* In the project directory, create a virtual environment and activate it: run  ```virtualenv -p python3 venv-django``` then ```source venv-django/bin/activate```
* Install requirements ```pip install -r requirements_devel.txt```
* Serve the application ```python manage.py runserver```
* In the browser, visit, ```http://127.0.0.1:8000/en/software/features```
* The Landing Page you see should look the screenshots shared below.

#### Any background context you want to provide?
* The background Image was changed because the current image was more masculine and the case here is to make it more appealing to both males and females.
#### What are the relevant pivotal tracker stories?
[#159236193](https://www.pivotaltracker.com/story/show/159236215)

#### Screenshots (if appropriate)

Re-designed Landing Pages
<img width="1677" alt="screen shot 2018-08-01 at 08 38 31" src="https://user-images.githubusercontent.com/39955231/43503146-ae483804-9566-11e8-8a14-8b57c254a5ca.png">

<img width="1673" alt="screen shot 2018-08-01 at 08 38 59" src="https://user-images.githubusercontent.com/39955231/43503151-b160cccc-9566-11e8-99c7-58d6c7fb3b59.png">
